WordSearch.java

```diff
@@ -0,0 +1,43 @@
+public class WordSearch {
+
+    //Time Complexity: O(MXN)
+    //Space Complexity: length of the word to search which is constant
```

Comment:
it is not constant. There can be duplicate letters as well. So that would use some space in recursive stack